File: LanguAI.Backend/LanguAI.Backend/Core/Models/Card.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace LanguAI.Backend.Core.Models;

public class Card
{
    [Required]
    public int Id { get; set; }

    [Required]
    public string WordInNativeLanguage { get; set; }

    [Required]
    public string WordInLearningLanguage { get; set; }

    [Required]
    public int CardListId { get; set; }

    [ForeignKey(nameof(CardListId))]
    public virtual CardList CardList { get; set; }
}

File: LanguAI.Frontend/src/api/models/card.ts
/* tslint:disable */
/* eslint-disable */
import { CardList } from '../models/card-list';
export interface Card {
  cardList?: CardList;
  cardListId: number;
  id: number;
  wordInLearningLanguage: string;
  wordInNativeLanguage: string;
}
